app/controllers/concerns/idv_session.rb

```diff
@@ -12,12 +12,37 @@ def confirm_idv_session_started
     redirect_to idv_session_url unless idv_session[:params].present?
   end
 
+  def confirm_idv_attempts_allowed
+    if idv_attempter.exceeded?
+      flash[:error] = t('idv.errors.hardfail')
+      redirect_to idv_fail_url
+    elsif idv_attempter.reset_attempts?
+      self.idv_attempts = 0
+    end
```

Comment:
Can you write a test for this, please? Code Climate is saying the elsif is not tested.